components/common/NotificationBox/NotificationItem.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <nuxt-link
+    :to="`/${urlPrefix}/gallery/${event.nft.id}`"
+    class="is-flex notify-item">
+    <img
+      v-if="avatar"
+      :src="avatar"
+      :alt="event.nft.name"
+      width="50"
+      height="50"
+      class="border image-size" />
+    <img
+      v-else
+      src="/placeholder.webp"
+      width="50"
+      height="50"
+      class="border image-size" />
```

Comment:
does `image-size` work here?
I would use something more standard: wrap it into 'image is-48x48`